### PR TITLE
Release/0.12.0

### PR DESCRIPTION
**Pull request by oguzhanunlu**

### CODE REVIEW

File: provisioning/roles/docker/files/docker-compose.yml [hunk 1]
```diff
@@ -38,6 +38,10 @@ services:
       - "5601:5601"
     depends_on:
       - elasticsearch
+    logging:
+      options:
+        max-size: "1M"
```
Comment (dilyand) [inline provisioning/roles/docker/files/docker-compose.yml:43]: Why are some configured with `max-size: "1M"` and others with `max-size: "10M"`?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,12 @@
+Version 0.12.0 (2020-12-15)
```
Comment (dilyand) [inline CHANGELOG:1]: Did we agree to include the version in the commit message, like "Prepare for 0.12.0 release"?

**Review by dilyand — APPROVED** [review]
Looks great @oguzhanunlu !

File: provisioning/roles/docker/files/docker-compose.yml (hunk 1 above)
Comment (oguzhanunlu) [reply provisioning/roles/docker/files/docker-compose.yml:43]: I thought not all logs are important but I can make them equal to be safe.

File: CHANGELOG (hunk 2 above)
Comment (oguzhanunlu) [reply CHANGELOG:1]: not sure, let me double check and modify if so

**Pull request merged**